### DISCUSSION TITLE
Compound device SCPI commands

### DISCUSSION THREAD

**Discussion by MarcelNeum** [jankae/LibreVNA#196] [Q&A]
Hi,
I wanted to ask if there is or will be a way to control/create a compound device of two libre VNAs via SCPI commands?
I just saw this feature in the User manual, but didnt find it in the SCPI Programming guide. I'm very interested in this feature but i want to make & save automated meassurements, which isnt very practical via the UI.
Best regards Marcel

**Comment by jankae** [comment]
Hi Marcel,
there are no SCPI commands to create the compound device but you only need to do that once. Afterwards you can connect to it just as you would with a single LibreVNA. All the SCPI commands for controlling the sweep/acquisition/extracting trace data should still work.

**Reply by MarcelNeum** [reply]
Thank you for the quick response, thats great.
